api/v1alpha1/profilecatalogsource_types.go

```diff
@@ -67,6 +67,16 @@ type ProfileCatalogEntry struct {
 
 // ProfileCatalogSourceStatus defines the observed state of ProfileCatalogSource
 type ProfileCatalogSourceStatus struct {
+	ScannedRepositories []ScannedRepository `json:"scannedRepositories,omitempty"`
+}
+
+// ScannedRepository contains the list of repositories that have been scanned and
+// what tags have bene processed
+type ScannedRepository struct {
+	// URL is the repository URL
+	URL string `json:"url,omitempty"`
+	// Tags is the list of tags that have been scanned
+	Tags []string `json:"tags,omitempty"`
```

Comment:
I feel like this could have a better name, LMK if anyone has one :smile:

Reply:
Looks good enough to me, reflects what it is :-)